swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, age=None, condition=None, category=None):
+        self.age = age if age is not None else 0
+        self.condition = condition if condition is not None else 0.0
+        category = category if category is not None else ""
+        self.category = category
```

Comment:
Since strings are an immutable data type it is safe to default the category as category="". You don't need to set `category` to None and then write conditions to check if it's Falsey and then set it to an empty string. 

The same applies for `condition` and `age` too. We can have condition and age both set to 0 since integers are also immutable. 

If any values are passed in for category and condition then the instance attributes will get those values instead of the defaults here. 

Since you're not needing to check if category and condition are Falsey and then setting them to "" or 0.0, the function body will be shorter:

```python
 def __init__(self, category="", condition=0):
        self.category = category
        self.condition = condition
        self.age = age
```